StreamChatUITestsAppUITests/Tests/MessageList_Tests.swift

```diff
@@ -162,7 +162,7 @@ final class MessageList_Tests: StreamTestCase {
         AND("participant deletes the message: '\(message)'") {
             participantRobot
                 .waitForNewMessage(withText: message)
-                .chill(duration: 2)
+                .chill(duration: 2000)
```

Comment:
If we are waiting here for the new message come, why do we need to wait an additional time? 🤔